pkg/info/infopb/rpc.proto

```diff
@@ -23,12 +23,12 @@ option (gogoproto.goproto_sizecache_all) = false;
 /// Info represents the API that is responsible for gathering metadata about the all APIs supported by the component.
 service Info {
     /// Info returns the metadata (Eg. LabelSets, Min/Max time) about all the APIs the component supports.
-    rpc Info(InfoRequest) returns (InfoResponse);
+    rpc Info(InfoReq) returns (InfoResp);
```

Comment:
First time I see this practice. While it helps with shorter name it violates coding style guide + consistency. Do we think it's worth it?

Reply:
If I don't rename it then a warning message will also be printed in our documentations as I mentioned [here](https://github.com/thanos-io/thanos/pull/4282#issuecomment-860240272). I was not able to find a way to remove them. @GiedriusS has suggested better names `InfoAPIRequest` and `InfoAPIResponse`.

Reply:
This is because our pacakge is `thanos`. Do you mind changing this to `thanos.info`?